matchzoo/auto/tuner/callbacks/load_embedding_matrix.py

```diff
@@ -1,13 +1,39 @@
 from matchzoo.engine.base_model import BaseModel
-from .callback import Callback
+from matchzoo.auto.tuner.callbacks.callback import Callback
 
 
 class LoadEmbeddingMatrix(Callback):
     """
     Load a pre-trained embedding after the model is built.
 
+    Used with tuner to load a pre-trained embedding matrix for each newly built
+    model instance.
+
     :param embedding_matrix: Embedding matrix to load.
 
+    Example:
+
+        >>> import matchzoo as mz
+        >>> model = mz.models.ArcI()
+        >>> prpr = model.get_default_preprocessor()
+        >>> data = mz.datasets.toy.load_data()
+        >>> data = prpr.fit_transform(data, verbose=0)
+        >>> embed = mz.datasets.toy.load_embedding()
+        >>> term_index = prpr.context['vocab_unit'].state['term_index']
```

Comment:
you use `context` instead of `state` in #701 , correct?

Reply:
These are two different PR into 2.2, will do a full replacement from state to context in the next version.

Reply:
@uduse ok